### PR TITLE
Update message input styles.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review. You'll definitely want to check out the branch to test it. There's a lot of cleanup in here too.

### CODE REVIEW

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js [hunk 1]
```diff
@@ -215,143 +210,154 @@ class PlatformInput extends Component<PlatformInputProps & OverlayParentProps, S
...
+          />
+        )}
+        <Kb.Box
+          style={collapseStyles([
+            styles.inputWrapper,
+            {
+              backgroundColor: this.props.isEditing ? globalColors.yellow3 : globalColors.white,
+              borderColor: this.props.explodingModeSeconds
+                ? globalColors.black_75
+                : this.props.isEditing
```
Comment (buoyad) [inline shared/chat/conversation/input-area/normal/platform-input.desktop.js]: don't think this second ternary is needed

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js [hunk 2]
```diff
@@ -215,143 +210,154 @@ class PlatformInput extends Component<PlatformInputProps & OverlayParentProps, S
...
+              ])}
+            >
+              {this.props.explodingModeSeconds ? (
+                <Kb.Text type="BodyTinyBold" style={styles.time}>
+                  {formatDurationShort(this.props.explodingModeSeconds * 1000)}
+                </Kb.Text>
+              ) : (
+                <Kb.Icon
+                  className="timer"
+                  onClick={this.props.isEditing ? undefined : this._toggleShowingMenu}
```
Comment (buoyad) [inline shared/chat/conversation/input-area/normal/platform-input.desktop.js]: is this ternary needed? this doesn't render if `!isEditing`

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js [hunk 3]
```diff
@@ -215,143 +210,154 @@ class PlatformInput extends Component<PlatformInputProps & OverlayParentProps, S
...
+            rowsMin={1}
+            rowsMax={10}
+            onKeyDown={this._onKeyDown}
+            onEnterKeyDown={this._onEnterKeyDown}
+          />
+          {flags.explodingMessagesEnabled &&
+            this.props.isExploding &&
+            !this.props.isEditing &&
+            !this.state.hasText && (
+              <Kb.Icon
```
Comment (buoyad) [inline shared/chat/conversation/input-area/normal/platform-input.desktop.js]: this is the boom placeholder icon? :sheep: might be nice to add a comment

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js (hunk 1 above)
Comment (adamjspooner) [reply shared/chat/conversation/input-area/normal/platform-input.desktop.js]: It was left over from testing colors. Thanks!

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js (hunk 2 above)
Comment (adamjspooner) [reply shared/chat/conversation/input-area/normal/platform-input.desktop.js]: Another testing remnant. Thanks!

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js (hunk 3 above)
Comment (adamjspooner) [reply shared/chat/conversation/input-area/normal/platform-input.desktop.js]: Done!

**Pull request merged**